pkg/authentication/basic/htpasswd.go

```diff
@@ -5,18 +5,21 @@ import (
 	"crypto/sha1" // #nosec G505
 	"encoding/base64"
 	"encoding/csv"
-	"fmt"
 	"io"
 	"os"
+	"sync/atomic"
+	"unsafe"
```

Comment:
Yikes, do we really need to use unsafe? 😨

Reply:
Removed the use of `unsafe`